libcaf_core/caf/flow/observable_builder.hpp

```diff
@@ -78,15 +78,15 @@ class CAF_CORE_EXPORT observable_builder {
 
   observable_builder& operator=(const observable_builder&) noexcept = default;
 
-  /// Creates a @ref generation that emits values by repeatedly calling
-  /// `generator.pull(...)`.
+  /// Creates a \link caf::flow::generation \endlink that emits values by
+  /// repeatedly calling `generator.pull(...)`.
```

Comment:
Please stick to javadoc style, i.e., `@` prefix. Same for the other occurrences in this file.